app/src/main/java/com/example/tasky/itemdetail/domain/EventRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.tasky.itemdetail.domain
+
+import com.example.tasky.core.domain.model.AgendaItem
```

Comment:
This should be in the feature package related to the event detail

Not necessary on the core

Reply:
Not sure if you are talking about EventRepository or AgendaItem

Reply:
I think what he meant was the agenda feature, not the event feature (since there is no event feature). But it's correct for both, since Tasky has 2 features (auth and agenda) and `EventRepository` and `AgendaItem` are both only needed in the agenda feature, they should go in there